### PR TITLE
(fix 1771) added url prefix

### PR DESCRIPTION
**Pull request by edewit**
fix for issue https://github.com/openshiftio/openshift.io/issues/1771

### CODE REVIEW

**Comment by fabric8cd** [comment]
@edewit fabric8/fabric8-ui:SNAPSHOT-PR-2355-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2355-fabric8-ui.badger.fabric8.io

**Comment by joshuawilson** [comment]
please fix your commit syntax.
`fix(launcher): add a url prefix to the forge api`
something like that

**Comment by vpavlin** [comment]
The snapshot works

File: src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ export class InputActionDialog {
       } else if (!jenkinsNamespace) {
         console.log("Warning no jenkinsNamespace on the Build!")
       } else {
-        url = pathJoin(forgeUrl, "/services/jenkins/", jenkinsNamespace, url);
+        url = pathJoin(forgeUrl, "/api/openshift/services/jenkins/", jenkinsNamespace, url);
```
Comment (joshuawilson) [inline src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts]: the change looks fine but the commit syntax is wrong

Comment (gastaldi) [reply src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts]: I think we can fix the commit syntax when squashing and merging this PR?

Comment (joshuawilson) [reply src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts]: that will work too

Comment (gastaldi) [reply src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts]: How about `fix(#1771): Add a URL prefix to forge API`? Does it have to be `fix(launcher)`?

Comment (joshuawilson) [reply src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.ts]: the thing in parens is the subject, not the issue number. I could be wizard or build or planner. It sorts the changes by subject.

**Comment by joshuawilson** [comment]
why didn't you wait for the `continuous-integration/jenkins/pr-merge` to go green?

**Comment by gastaldi** [comment]
It was already green before I added the second commit rebasing the PR with the latest master. 

When I added the second commit, the build was taking forever to complete and I assumed it was hung and proceeded.

**Comment by joshuawilson** [comment]
You could have followed the `details` link to see what was going on. The only assurances we have are the checks in the pipeline. Please do NOT merge unless they are green.

**Pull request merged**